tip-calculator/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tip Calculator</title>
+    <link href="https://fonts.googleapis.com/css?family=Jura&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <h1>Hack your tips!</h1>
+    <p>Rather than try and figure out what kind of tip to leave using a percentage, just rate your experience!</p>
+    <div class="app">
+        <div class="calculator">
+            <header>
+                <h1>Tip Hacker</h1>
+                <form>
+                    <h3>What's the bill?</h3>
+                    <input id="bill" type="text" placeholder="Let's hack it!">
+                    <h3>How was this experience for you?
+                        <select id="rating">
+                            <option>Be real, how was it?</option>
+                            <option value="0.0">0% - I should have stayed home.</option>
+                            <option value="0.1">10% - It was better than cooking for myself.</option>
+                            <option value="0.15">15% - They should entertain other work options...</option>
+                            <option value="0.20">20% - Service was as expected, looking forward to returning.</option>
+                            <option value="0.25">25% - I can't wait to come back, and share this joint with more people!</option>
+                            <option value="0.3">30% - This was a moving experience for me... I'll just leave it at that, see you soon!</option>
+                            <option value="0.5">50% - I'm just here to make someone's day! Cheers!</option>
+                        </select>
```

Comment:
consider letting the user put the amount themselves with an input field... i think it is better ux

Reply:
I think for the tip having an option like above is a good idea, that way the user wouldn't have to guess about what to input, using whole numbers or decimals or %.

Reply:
@jcarrera94 It is something I would have done with more time, but I wanted something fun and easy for users to utilize out of the gate. As a person who has worked in the service industry for 15+ years, this seemed like a simple way to put some flare on this project. 

Thanks @Andacanaver